app/assets/javascripts/steps.js

```diff
@@ -147,14 +147,6 @@
         $('.add-change-note-description--textarea').val('');
         $('.add-change-note--details').removeAttr('open');
       });
-    },
-
-    bindCancelAddSecondaryLink: function() {
```

Comment:
This is being called from line 17. Can you remove it from there too?

Reply:
Good spot, updated.